Documentation/project-docs/platform-names.md

```diff
@@ -0,0 +1,61 @@
+# Platform Names
+
+Over the years, we've accumulated quite a few names, platforms, and components that can make it hard for folks (including us) to understand what we're referring to. This document has a list that will help to qualify what we mean by what.
+
+This will also list some aliases. As you'll see the aliases aren't always correct -- they are merely listed to help you find the better and less confusing terminology.
+
+## .NET Framework
+
+**Also referred to as**: Desktop, full framework, in-box framework
+
+This refers to the .NET Framework that shipped in 2002. It's the main framework folks target today and allows you to build a wide variety of applications, such as WinForms, WPF, ASP.NET, and command line tools.
+
+The .NET Framework was designed to run on Windows only. Some versions of the .NET Framework come pre-installed with Windows, some require to be installed. However, in both cases the .NET Framework is a system-wide component.
+
+## .NET Core
+
+**Also referred to as**: UWP, ~~Store~~
+
+Originally, .NET Core was the identifier we used to describe the surface area Windows 8 store applications can use. When we designed the API set, we wanted to create a foundation for .NET where portability is a first class concern for the layering and componentization. For more details, read [this blog post](http://blogs.msdn.com/b/dotnet/archive/2014/12/04/introducing-net-core.aspx).
+
+Today, .NET Core no longer only powers store applications. .NET Core is the name for the open source, cross-platform stack that ASP.NET Core and UWP applications are built on. The stack includes a set of framework libraries (CoreFX), a JIT based runtime (CoreCLR), an AOT based runtime (CoreRT), and a set of tooling (such as the dotnet CLI).
```

Comment:
Yes, I noticed, but since people usually read top to bottom, it might still be confusing until the reader reaches the next section. Perhaps turn it into a link that points to the relevant section?